src/BUILD.bazel

```diff
@@ -197,6 +197,23 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "stacktrace",
+    srcs = ["platform/stacktrace.cpp"],
+    hdrs = ["platform/stacktrace.h"],
+    copts = COPTS + [
+        # "-DQTOX_USE_LIBUNWIND",
+        "-Iexternal/libunwind/include",
```

Comment:
Does Bazel require these even when the includes are ifdef'd out?

Reply:
It's broken and I want to fix it soon, so I just commented out as much as I needed to get it to build.